### ISSUE TITLE
Account verification mail not received

### ISSUE THREAD

**Issue by shravanipl** [walmartlabs/walmart-api#9]
Hi,
I am trying to create account for Developer access to the Walmart API. After entering the all the details, its showing that Confirmation mail is sent to the email address, but I have not received any mail. Tried multiple times but still facing the same issue.

**Comment by ussefad** [comment]
Who are you?

**Comment by gp7029** [comment]
@ussefad The user posted a legitimate question.

**Comment by howitworks18** [comment]
Same issue ^^ attempted multiple times, no email

**Comment by kalittles** [comment]
I have the exact same issue. I've tried multiple times.

**Comment by mosabua** [comment]
I will reach out internally at Walmart to find out and get them to reach out to you each. Please ensure your github profile has an email we can contact you at.

**Comment by gp7029** [comment]
@mosabua Thanks! Really appreciated

**Comment by angoothachap** [comment]
I have the exact same problem

**Comment by paullovachy** [comment]
Same problem for me as well.

**Comment by kohlbrr** [comment]
Same problem for me as well - attempted resend twice

**Comment by dylancwood** [comment]
Hi all, a colleague just pointed me to this repo and issue. While I am not a member of the team that maintained this repository or the developer.walmart.com experience, I do work at Walmart Labs. 

The developer portal is definitely not working as designed, and there appears to be no one maintaining it. I will work to either rectify the situation or have the portal taken offline.

There is a new portal being brought online over the next months at walmart.io. It is still very much a work in progress, and I do not recommend that anyone try to use it just yet. 

I will report back here by the end of the week with an update.

Thanks for your interest in legitimately using Walmart APIs!

**Comment by gp7029** [comment]
@dylancwood Thanks for the update. Much appreciated!

**Comment by shravanipl** [comment]
@dylancwood Thanks for the update.

**Comment by mosabua** [comment]
@cyuan2-wmt please ensure you help all the users here and then close this issue. 

Everyone keep in mind .. this repo/api is now deprecated.

**Comment by cyuan2-wmt** [comment]
Hi all,
If you go to https://developer.walmartlabs.com/, you will see the following disclaimer:
Registration for new Walmart API account is currently closed.
That's why the registration emails are not sent. 
We are targeting end of the year to re-open the registration. Please come back then.

**Comment by kalittles** [comment]
I'm the founder of a startup and utilization of Walmart's API is part of our product - it advertises for WalMart and drives customer engagement directly to Walmart.com.

Our only other solution is to integrate Amazon into our platform.
Is there really zero possibility of getting an API key? Because if I don't know by the end of the month, I'll simply tell our stakeholders we're dropping Walmart support entirely.

Thanks.

**Comment by angoothachap** [comment]
I'm not even able to use the Affiliate API with all the required additional headers. The Swagger API is also not helpful - uses a publisherId that's nowhere on the specs. Where is the on boarding guide as mentioned in https://walmart.io/docs/affiliate/introduction

**Comment by dylancwood** [comment]
@kalittles: Let me see if we can come up with a more manual process while the new portal is set up.

@angoothachap: We've encountered the same issues, and are pushing the walmart.io team to improve things. Will keep you posted.

**Comment by angoothachap** [comment]
ok, thanks @dylancwood.

**Comment by dylancwood** [comment]
@kalittles: I reached out on LinkedIn with email addresses that will be useful for you.
@angoothachap: I emailed you the same info.

Thanks.

**Comment by angoothachap** [comment]
@dylancwood I am not sure what email ID you used. I didn't receive anything on email registered with GitHub. 
On a side note, I'm willing to offer help in writing/fixing code for the API. Do let me know.

Thanks!

**Comment by kalittles** [comment]
@dylancwood 

I sent over an email - were you able to read over it?
If not, I can send over another with more information.

Thanks!

**Comment by ussefad** [comment]
got it

**Comment by Aldo-Walmart** [comment]
Hi everyone! this is Aldo from Walmart. As a reminder, our API program is at capacity and closed for new registrations. We are making some changes that will improve our API platform. Will open up new registrations in the coming months once the platform is ready.

Thank you for your patience.
Aldo

**Comment by dishashetty** [comment]
Hi, is the API key available now? is the registration still closed? I have created an account in impact and walmart both but not received my API access key.

**Comment by dylancwood** [comment]
Hi @dishashetty . I don't think that walmart.io is ready for public consumption yet. If you have an urgent need for access to Walmart APIs, please reach out to me directly. My email is in my profile

**Comment by Aldo-Walmart** [comment]
@dishashetty .. thank you for signing up. We are still in the middle of transition which will have to put registration on hold a while longer. Please reach me directly at abukit-at-walmart.com and I can manually create an API account for you.

**Comment by Aldo-Walmart** [comment]
> Hi @dishashetty . I don't think that walmart.io is ready for public consumption yet. If you have an urgent need for access to Walmart APIs, please reach out to me directly. My email is in my profile

Dylan .. do you mind connect with me? you have a Gmail email address. I'm not sure if you're working for Walmart. thx.

**Comment by dishashetty** [comment]
@Aldo-Walmart  I have sent you a n email on abukit@walmart.com

**Comment by Aldo-Walmart** [comment]
replied!

Do you mind removing my email address from your post above? You put the correct syntax which is open for crawlers to grab. Thank you.

**Comment by dishashetty** [comment]
@Aldo-Walmart  deleted!
